lib/preload.js

```diff
@@ -9,8 +9,8 @@ const tar = Promise.promisifyAll(require('tar-stream'))
 const fs = Promise.promisifyAll(require('fs'))
 const errors = require('resin-errors')
 const request = require('request-promise')
-const visuals = require('resin-cli-visuals')
 const getPort = require('get-port')
+const EventEmitter = require('events')
```

Comment:
Oh man, I never knew that worked!
I always did `var EventEmitter = require('events').EventEmitter`. TIL.